.github/workflows/ci.yaml

```diff
@@ -68,3 +68,5 @@ jobs:
           file: coverage.xml
           fail_ci_if_error: True
           verbose: True
+        if: ${{ github.repository == 'OpenFreeEnergy/openfe'
```

Comment:
more of a flair thing - would you be happy to move this up under the `name` section? in my opinion makes it easier to scan through and see which steps have controls on them.